src/core.js

```diff
@@ -161,6 +161,9 @@ export default class Base extends EventEmitter {
   getUserInfo(token, cb) {
     return webAPI.getUserInfo(this.id, token, cb);
   }
```

Comment:
the `beautifier` didn't add a new line here. Is that hook working?

Reply:
it is working, but it doesn't add extra lines. I just added one